GitUI/Translation/English.xlf

```diff
@@ -3568,6 +3568,10 @@ You can unset the template:
         <source>Interactive Add</source>
         <target />
       </trans-unit>
+      <trans-unit id="modifyCommitMessageButton.Text">
+        <source>Modify this special commit m&amp;essage</source>
+        <target />
+      </trans-unit>
```

Comment:
The TranslationApp seems to ignore ToolTips. Must I really create a TranslationString by hand?